utils/file.go

```diff
@@ -67,10 +67,22 @@ func Mkdir(dirName string) error {
 }
 
 func MkTmpdir() (string, error) {
-	tempDir := filepath.Join(os.TempDir(), GenUniqueID(32))
+	tempDir, err := ioutil.TempDir(common.DefaultTmpDir, ".DTmp-")
+	if err != nil {
+		return "", err
+	}
 	return tempDir, os.MkdirAll(tempDir, os.ModePerm)
 }
 
+func MkTmpFile() (*os.File, error) {
+	if !IsDir(common.DefaultTmpDir) {
```

Comment:
You can mkdir for DefaultTmpDir at boot module to ensure this dir existence and its type

Reply:
OK